Resources/Private/Templates/Page/Chapter.html

```diff
@@ -26,9 +26,7 @@
 					</header>
 				</neos:contentElement.wrap>
 
-				<f:if condition="{node.properties.chapterImage}">
-					<media:image asset="{node.properties.chapterImage}" class="pull-right" alt="{node.properties.chapterImage.alternativeText}" title="{node.properties.chapterImage.title}" />
-				</f:if>
+				<media:image image="{node.properties.chapterImage}" class="pull-right" alt="{node.properties.chapterImage.alternativeText}" title="{node.properties.chapterImage.title}" />
```

Comment:
In `ChapterMenu.html` and `CarouselItem.html` there is an `else` part, too so it would change behavior